bot/cogs/antimalware.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from discord import Message, NotFound
+from discord.ext.commands import Bot, Cog
+
+from bot.constants import AntiMalware as AntiMalwareConfig, Channels
+
+log = logging.getLogger(__name__)
+
+
+class AntiMalware(Cog):
+    """Cog providing anti-malware behavior."""
```

Comment:
Original wasn't very helpful. It's pretty obvious it's a cog and we can already see its name is "AntiMalware".

```suggestion
    """Delete messages which contain attachments with non-whitelisted file extensions."""
```